about/list_of_features.rst

```diff
@@ -0,0 +1,510 @@
+- Support for advertisements using third-party modules.
+
+Mixed reality
+^^^^^^^^^^^^^
+
+- Support for ARKit on iOS out of the box.
+- Popular VR headsets like the Oculus Quest and HTC Vive are supported thanks to plugins.
+
+GUI system
+^^^^^^^^^^
+
+Godot's small distribution size can make it a suitable alternative to frameworks
+like Electron or Qt.
+
+**Nodes:**
+
+- Buttons.
+- Checkboxes, check buttons, radio buttons.
+- Text entry using LineEdit (single line) and TextEdit (multiple lines).
+- Dropdown menus using PopupMenu and OptionButton.
+- Scrollbars.
+- Labels.
+- RichTextLabel for :ref:`text formatted using BBCode <doc_bbcode_in_richtextlabel>`.
+- Trees (can also be used to represent tables).
+- Containers (horizontal, vertical, grid, center, margin, draggable splitter, ...).
+- Controls can be rotated and scaled.
+
+**Sizing:**
+
+- Anchors to keep GUI elements in a specific corner, edge or centered.
+- Containers to place GUI elements automatically following certain rules.
+   - :ref:`Stack <class_BoxContainer>` layouts.
+   - :ref:`Grid <class_GridContainer>` layouts.
+   - :ref:`Margin <class_MarginContainer>` and :ref:`centered <class_CenterContainer>`
+     layouts.
+   - :ref:`Draggable splitter <class_SplitContainer>` layouts.
+- Scale to multiple resolutions using the ``2d`` or ``viewport`` stretch modes.
+- Support any aspect ratio using anchors and the ``expand`` stretch aspect.
+
+**Theming:**
+
+- Built-in theme editor.
+   - Generate a theme based on the current editor theme settings.
+- Procedural vector-based theming using :ref:`class_StyleBoxFlat`.
+   - Supports rounded/beveled corners, drop shadows and per-border widths.
+- Texture-based theming using :ref:`class_StyleBoxTexture`.
+
+Animation
+^^^^^^^^^
+
+- Direct kinematics and inverse kinematics.
+- Support for animating any property with customizable interpolation.
+- Support for calling methods in animation tracks.
+- Support for playing sounds in animation tracks.
+- Support for Bézier curves in animation.
+
+Formats
+^^^^^^^
+
+- Scenes and resources can be saved in :ref:`text-based <doc_tscn_file_format>` or binary formats.
+   - Text-based formats are human-readable and more friendly to version control.
+   - Binary formats are faster to save/load for large scenes/resources.
+- Read and write text or binary files using :ref:`class_File`.
+   - Can optionally be compressed or encrypted.
+- Read and write :ref:`class_JSON` files.
+- Read and write INI-style configuration files using :ref:`class_ConfigFile`.
+   - Can (de)serialize any Godot datatype, including Vector, Color, ...
+- Read XML files using :ref:`class_XMLParser`.
+- Pack game data into a PCK file (custom format optimized for fast seeking),
+  into a ZIP archive, or directly into the executable for single-file distribution.
+- :ref:`Export additional PCK files<doc_exporting_pcks>` that can be read
+  by the engine to support mods and DLCs.
+
+Miscellaneous
```

Comment:
"Low-level access to Server API which allows users to avoid the overhead from the scenetree"